sympy/solvers/tests/test_ode.py

```diff
@@ -2287,13 +2287,17 @@ def test_nth_linear_constant_coeff_variation_of_parameters_simplify_False():
     assert sol_simp != sol_nsimp
     # /----------
     # eq.subs(*sol_simp.args) doesn't simplify to zero without help
-    zero = checkodesol(eq, sol_simp, order=5, solve_for_func=False)[1]
+    (t, zero) = checkodesol(eq, sol_simp, order=5, solve_for_func=False)
     # if this fails because zero.is_zero, replace this block with
     # assert checkodesol(eq, sol_simp, order=5, solve_for_func=False)[0]
-    assert not zero.is_zero and zero.rewrite(exp).simplify() == 0
+    assert not zero.is_zero and zero.rewrite('exp').simplify() == 0
```

Comment:
Why the use of quotes on `exp`?

Reply:
This code has gone back and forth, so I must have copied the line with quotes when I restored it (and not imported exp when trying it out). Will update.

Reply:
I checked...exp is imported in this file.

Reply:
Yes, was probably from when I tried it out by hand in the shell... I've changed it (and the as_coeff-stuff and probably some more). PR soon if the tests pass locally...